examples/purchaseTesterTypescript/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "purchasetester",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "ios": "react-native run-ios",
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@react-navigation/bottom-tabs": "^6.0.9",
+    "@react-navigation/native": "^6.0.6",
+    "@react-navigation/native-stack": "^6.2.5",
+    "react": "17.0.2",
+    "react-native": "0.66.4",
+    "react-native-purchases": "^4.5.0",
```

Comment:
@vegaro Thoughts on doing this in a separate PR? 😁 I started messing around with this and it got weird 😛 I think it would be nice to see a separate diff and for historical purposes 🤷‍♂️